ppa/archive/management/commands/gale_import.py

```diff
@@ -82,6 +82,12 @@ def handle(self, *args, **kwargs):
         if not (kwargs["ids"] or kwargs["csv"]):
             raise CommandError("A list of IDs or CSV file for is required for import")
 
+        # error handling in case user forgets to specify csv file correctly
+        if kwargs.get('id') and len(kwargs['ids']) == 1 and kwargs['ids'][0].endswith('.csv'):
+            self.stdout.write(self.style.WARNING(
+                '%s is not a valid id; did you forget to specify -c/--csv?' % kwargs['ids'][0]))
+            return
+
```

Comment:
is it `id` or `ids` (or both)? first condition in the `if` has `id`. also i sometimes do `if 'id' in kwargs` because i think it reads slightly better, but idk if that's python best practice or anything

Reply:
good catch, thank you! pretty sure `if 'id' in kwargs' is more pythonic, will revise